export_helpers/containerize_readme.md

```diff
@@ -20,6 +20,13 @@ Install on openSUSE 13.2:
 
 Log out and in again to refresh the user's group.
 
+## Set Up
+
+We've included a rake task to setup your new containerized application. Please
+run it before starting docker-compose.
+
+    rake setup
```

Comment:
@cornelius I liked the idea of having more control through the API so it was possible to check when a container was running or not. Here is a version with the shell script, definitely more simple but I still don't like to have to wait for the db on a loop, specially when it cannot connect for some issue.

https://github.com/SUSE/machinery/pull/1294